test/test_transform.cpp

```diff
@@ -2074,4 +2074,316 @@ TEST(Transform_ArrayEraseVsArrayErase)
     server->integrate_next_changeset_from(*client_5);
 }
 
+TEST(Transform_RSYNC_143)
```

Comment:
I'll add generating a new test corpus as a goal of the RSYNC-143 server mitigation scope